### DISCUSSION TITLE
Wondering how many users uninstalled uBO recently :)

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2932] [General]
https://www.youtube.com/watch?v=Z6zZFj9eE60

**Comment by gorhill** [comment]
Well if we go by the [calculations I made there](https://github.com/uBlockOrigin/uBlock-issues/discussions/2854#discussioncomment-7223833) but using the [current stats shown at jsDelivr](https://www.jsdelivr.com/package/gh/gorhill/uBlock?tab=stats), with 50,853,665 requests we get 88.1M active installations, up from 83.7M in early October. But then it could just be that more people are clicking the "Purge all caches" button, this would also cause `assets.json` to be fetched more often.
